### ISSUE TITLE
Documentation: Webots.cloud

### ISSUE THREAD

**Issue by BenjaminDeleze** [cyberbotics/webots#4678]
We should create some documentation about Webots.cloud and integrate it in the user guide because:

- We are speaking of `webots.cloud` in the docs without explaining what it is.
- We have documentation about webotsJS at different places: in the docs and in the wiki of the `webots.cloud` repo.

We currently have the following pages in the docs in the section `Web Interface`:
- `X3D and Web Scene`
- `Web Animation`
- `Web Streaming`
- `Web Simulation`

And in the wiki:
- `Setup of a simulation server`
- `Setup of a session server`
- `Checking out only a git folder`
- `Running Webots in a Docker container`
- `Setup a Webots project Repository`
- `Use Cases`

We should address two main issues in my opinion:
- Regroup all the docs in one place. And I think this place should be the documentation of webots. It is nice to have it directly on the wiki of the repo but it is less accessible and "out" of the normal doc.
- We cannot just transfer all the pages from the wiki to the docs because of the redundancy (even between pages of the wiki): for example between `Web simulation`, both wikis `Setup` pages and `Running Webots in a Docker container`.

Here is an idea of how to rearrange thinks:

`Web Interface`:
- `X3D and Web Scene`
- `Web Animation`
- `Web Streaming`
- `Webots.cloud` (presentation, how it works, feature)

`Web Server`:
- `Overview and Prerequisites`
- `Protocol and Quick Start`
- `Session Server`
- `Simulation Server`
- "Last steps": port rewrite, server display and startup procedure
- `Advanced configuration`: docker, checking out only a git folder, tips to increase performance
- `Setup a Webots Project Repository`
- `Use case`

So I would keep `X3D and Web Scene`, `Web Animation` and `Web Streaming` as is.

Then I would add a section `How to Embed a Web Scene in Your Website` and `Programming interface` to `Web Streaming`. The text of this section is already existing in `How to Embed a Web Scene in Your Website` of the `Web Simulation` page.

I would also split the `How to Embed a Web Scene in Your Website` of `X3D and Web Scene` to also have a `Programming interface section`.

I think it is a good idea to dismantle the `Web Streaming` page in smaller pages.

For the `Prerequisites`, `Session Server` and `Simulation Server` pages, we should merge the content of the wiki and the docs.

Once it is done, I would delete the wiki to ease the maintenance and just put a link to the docs (as it is on a different repo)
